dashboard/app/controllers/script_levels_controller.rb

```diff
@@ -186,7 +186,10 @@ def show
       end
     end
 
-    @body_classes = @level.properties['background']
+    lesson = @script.lesson_by_relative_position(params[:lesson_position].to_i)
```

Comment:
nit: small comment here? Particularly just to avoid confusion on why we're pulling background from the lesson vs the level, as this controller action runs on specific level pages.

Reply:
Good call.  Will also update the code to not make a new query when the lesson should already be cached.